### DISCUSSION TITLE
when applying i receive this error: We require applicants to use a school-issued email address to apply.

### DISCUSSION THREAD

**Discussion by 0xPix3l** [community/community#51035] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

iam applying for student pack and i received this error:
We require applicants to use a school-issued email address to apply.

Cairo university faculty of agriculture

- agr.cu.edu.eg
- stud.cu.edu.eg

**Comment by 0xPix3l** [comment]
any help?

**Reply by cMarteli** [reply]
I'm getting the same error. Tried submitting a ticket and GitHub closed it one day later without solving the problem.

**Comment by mudassarsial1** [comment]
### _**i also facing this problem**_

**Comment by samus-aran** [comment, marked as answer]
We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [I don't have a school-issued email address, what other official proof of enrollment is accepted?](https://github.com/orgs/community/discussions/111352#i-dont-have-a-school-issued-email-address)
- [What types of evidence does the Student Developer Pack application accept?](https://github.com/orgs/community/discussions/111352#what-types-of-evidence-does-the-student-developer-pack-application-accept)
- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

**Reply by balajirajput96** [reply]
Yes

**Comment by omralabiyd33-max** [comment]
import requests

# 1. حدد البيانات الخاصة بمشروعك
GITLAB_URL = "https://gitlab.com"  # أو رابط سيرفر GitLab الخاص بشركتك
PROJECT_ID = "YOUR_PROJECT_ID"    # رقم المشروع (Project ID)
MR_IID = "YOUR_MR_IID"            # رقم طلب الدمج (Merge Request IID)
PRIVATE_TOKEN = "YOUR_ACCESS_TOKEN"  # Personal Access Token بتاعك

# 2. إعداد الرابط والـ Headers
url = f"{GITLAB_URL}/api/v4/projects/{PROJECT_ID}/merge_requests/{MR_IID}/approve"

headers = {
    "PRIVATE-TOKEN": PRIVATE_TOKEN
}

# 3. إرسال الطلب
response = requests.post(url, headers=headers)

# 4. التحقق من النتيجة
if response.status_code == 201:
    print(" تم قبول طلب الدمج (Approved) بنجاح!")
    print(response.json())
elif response.status_code == 401:
    print(" خطأ في التوثيق: تأكد من صحة الـ Personal Access Token.")
elif response.status_code == 404:
    print(" لم يتم العثور على المشروع أو طلب الدمج. تأكد من الـ Project ID والـ MR IID.")
else:
    print(f"حدث خطأ ({response.status_code}):", response.text)

**Reply by omralabiyd33-max** [reply]
import requests

# 1. حدد البيانات الخاصة بمشروعك
GITLAB_URL = "https://gitlab.com"  # أو رابط سيرفر GitLab الخاص بشركتك
PROJECT_ID = "YOUR_PROJECT_ID"    # رقم المشروع (Project ID)
MR_IID = "YOUR_MR_IID"            # رقم طلب الدمج (Merge Request IID)
PRIVATE_TOKEN = "YOUR_ACCESS_TOKEN"  # Personal Access Token بتاعك

# 2. إعداد الرابط والـ Headers
url = f"{GITLAB_URL}/api/v4/projects/{PROJECT_ID}/merge_requests/{MR_IID}/approve"

headers = {
    "PRIVATE-TOKEN": PRIVATE_TOKEN
}

# 3. إرسال الطلب
response = requests.post(url, headers=headers)

# 4. التحقق من النتيجة
if response.status_code == 201:
    print(" تم قبول طلب الدمج (Approved) بنجاح!")
    print(response.json())
elif response.status_code == 401:
    print(" خطأ في التوثيق: تأكد من صحة الـ Personal Access Token.")
elif response.status_code == 404:
    print(" لم يتم العثور على المشروع أو طلب الدمج. تأكد من الـ Project ID والـ MR IID.")
else:
    print(f"حدث خطأ ({response.status_code}):", response.text)

**Comment by omralabiyd33-max** [comment]
Cairo LM
